src/components/Layout.tsx

```diff
@@ -104,50 +107,136 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
-                    Compounders
-                  </Menu.Item>
-                  <Menu.Item
-                    as={DropdownMenuItemLink}
-                    href={appLink("/yield/concentrators")}
-                  >
-                    Concentrators
-                  </Menu.Item>
-                </Menu.Items>
-              </Transition>
-            </Menu>
-          </div>
-        </header>
+                <Dialog.Panel className="relative h-full w-full max-w-xs overflow-y-auto border-l border-pink/10 bg-gradient-to-tr from-pink-600/40 to-orange-600/40 shadow-xl backdrop-blur-xl focus:outline-none">
+                  <div className="grid h-full grid-cols-1 grid-rows-[1fr,auto] gap-16 p-3">
+                    <div>
+                      <div className="relative grid grid-cols-[auto,min-content,min-content] grid-rows-1 gap-x-2">
+                        <NetworkSelector />
+
+                        <TxSettingsPopover />
+
+                        <button
+                          className="h-12 w-12 shrink-0 rounded bg-pink-900/40 p-3.5"
+                          onClick={() => setSidebarOpen(false)}
+                          type="button"
+                        >
+                          <FortIconClose className="h-full w-full fill-white" />
+                        </button>
+                      </div>
+
+                      <nav className="mt-3">
+                        <h1 className="px-3 pt-3 text-xs font-medium uppercase tracking-wider text-orange-400">
+                          Yield
+                        </h1>
+                        <div className="space-y-2 pt-2">
+                          <a
+                            href={appLink("/yield")}
+                            className={clsxm(
+                              "block rounded px-3 py-2.5 text-lg font-medium text-white/80",
+                              {
+                                "bg-gradient-to-r from-orange-400/20 to-orange-400/5 text-white ring-1 ring-inset ring-orange-400/20":
+                                  router.pathname === "/app/yield",
+                              }
+                            )}
+                          >
+                            Compounders
+                          </a>
+                          <a
+                            href={appLink("/yield/concentrators")}
+                            className={clsxm(
+                              "block rounded px-3 py-2.5 text-lg font-medium text-white/80",
+                              {
+                                "bg-gradient-to-r from-orange-400/20 to-orange-400/5 text-white ring-1 ring-inset ring-orange-400/20":
+                                  router.pathname ===
+                                  "/app/yield/concentrators",
+                              }
+                            )}
+                          >
+                            Concentrators
```

Comment:
What about the ability to disable the concentrator?

Reply:
Good catch I will get that back in.